server/plugins/renderer/renderer.module.js

```diff
@@ -237,9 +237,12 @@ internals.parseResponse = async (bcAppData, request, response, responseArgs) =>
     }
 
     const formattedRegions = {};
-    regionResponse.renderedRegions.forEach((region) => {
-        formattedRegions[region.name] = region.html;
-    });
+
+    if (typeof regionResponse.renderedRegions !== 'undefined') {
```

Comment:
I would recommend to use 
```suggestion
    if (!regionResponse.renderedRegions) {
```

in case if _**regionResponse.renderedRegions**_ comes as **null** from backend

Reply:
I generally prefer doing explicit checks as its safer and is not prone to errors like something being a boolean. Let me know if you feel strongly about this change request.

Reply:
I'm good with the changes. I approved your PR :) 

Just curious what kind of errors are you referring to?

Reply:
In this particular case the `renderedRegions` will never be boolean, but if it were then `if (regionResponse.renderedRegions)` would fail to do the right thing because `false` is a valid value.